src/main/java/mil/dds/anet/views/IndexView.java

```diff
@@ -35,4 +38,12 @@ public void setSecurityBannerColor(String securityBannerColor) {
 		this.securityBannerColor = securityBannerColor;
 	}
 
+	public String getDictionary() {
```

Comment:
Add a method that does whatever processing is needed for the view, i.e.:
```
public String getSerializedDictionary() {
  return StringEscapeUtils.escapeEcmaScript(jsonMapper.writeValueAsString(dictionary));
}
```
(with proper exception handling).